networks/typical/main.tf

```
@@ -13,51 +13,69 @@ provider "docker" {
 
 locals {
   number_of_nodes = var.number_of_nodes
-  node_indices    = range(local.number_of_nodes)
+  node_indices = range(local.number_of_nodes)
   more_args = join(" ", [
-    "--allow-insecure-unlock" # since 1.9.7 upgrade
+    "--allow-insecure-unlock",
+    # since 1.9.7 upgrade,
```

```suggestion
    "--allow-insecure-unlock", # since 1.9.7 upgrade
```
the comment feels not related to the line anymore, is it possible to put like this?

---

I will fix in a separate branch (used auto merge so as you approved, it merged).

---

fixed on https://github.com/ConsenSys/quorum-acceptance-tests/pull/121